nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -757,6 +757,16 @@ auth required pam_succeed_if.so uid >= 1000 quiet
      </warning>
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The option <option>services.nixosManual.showManual</option> has been removed (and the module).
+	 You can use the <command>nixos-help</command> command instead, without security issues with the module's implementation.
+	 This applies in particular to the installation images.
```

Comment:
The indentation here isn't right.

Reply:
Let's also reference what the security issue was https://github.com/NixOS/nixpkgs/issues/83157. It wasn't in the PR initially, which made it look like a vanity removal. We can also mention below that the `services.rogue` worked in a similar way, which is why it had to be removed for security.